pkgs/by-name/gz/gzdoom/package.nix

```diff
@@ -28,16 +28,18 @@
 
 stdenv.mkDerivation rec {
   pname = "gzdoom";
-  version = "4.13.2";
+  version = "4.14.0";
 
   src = fetchFromGitHub {
     owner = "ZDoom";
     repo = "gzdoom";
     rev = "g${version}";
     fetchSubmodules = true;
-    hash = "sha256-3nkdpJ3XO58YHtjVTwxdSdCL6CnMcih6mTnI7FXLm34=";
+    hash = "sha256-+gLWt1qBKl8xGK6sALnjqPuXcBexjWKbEkbRMFtLcbE=";
   };
 
+  patches = [ ./string_format.patch ];
```

Comment:
It would be nice if you could add a comment with what exactly this does and why it's needed.

Reply:
nvm, I completely missed that in the PR description.